multy_core/src/ethereum/ethereum_transaction.h

```diff
@@ -63,11 +70,10 @@ class EthereumTransaction : public TransactionBase
     EthereumTransactionFeePtr m_fee;
     EthereumTransactionSourcePtr m_source;
     EthereumTransactionDestinationPtr m_destination;
```

Comment:
Please add a comment:
User-visible destination, where all values are explicitly set by user, including token receiver address and token transfer amount.